src/domain/event/enrol/Enrol.tsx

```diff
@@ -24,7 +25,20 @@ import { saveChildEvents, justEnrolled } from '../state/EventActions';
 import ErrorMessage from '../../../common/components/error/Error';
 import Button from '../../../common/components/button/Button';
 
-const Enrol: FunctionComponent = () => {
+function check<D>(data: D, resultFactory: (data: D) => boolean): boolean {
+  return resultFactory(data);
+}
+
+function containsAlreadyJoinedError(
+  errors: ReadonlyArray<GraphQLError>
+): boolean {
+  return errors.some(
+    (error: GraphQLError) =>
+      error.extensions?.code === 'CHILD_ALREADY_JOINED_EVENT_ERROR'
```

Comment:
Maybe store this error code in constant's file? I think similar thing was done in `profile-ui`.